src/main/kotlin/uk/gov/justice/digital/hmpps/supportadditionalneedsapi/service/PlanCreationScheduleService.kt

```diff
@@ -22,6 +23,7 @@ class PlanCreationScheduleService(
   private val educationService: EducationService,
   private val needService: NeedService,
   private val eventPublisher: EventPublisher,
+  @Value("\${pes_contract_date:}") val pesContractDate: LocalDate,
```

Comment:
Not going to suggest we change this, but I was reading somewhere recently that there is some "advice" ("advice" in this context is most likely subjective and opinionated! 🤣 ) that spring devs should stop using `@Value` (I think because they end up being littered all over the place), and instead should use a data class or similar annotated with `@ConfigurationProperties`

Something like this (though this wasnt the "advice" I was reading)
https://www.baeldung.com/configuration-properties-in-spring-boot

Anyway, as you were 😁 , it's fine as it is 👍

Reply:
Ok I'll have a look at that maybe sometime - I'll add a ticket.